examples/crowd-funding/README.md

```diff
@@ -0,0 +1,237 @@
+*successful* and the creator can receive all the funds, including ones exceeding the funding
+target. Otherwise, the campaign is *unsuccessful* and contributors should be refunded.
+
+# Caveat
+
+Currently, only the owner of the campaign can create blocks that contain the `Cancel`
+operation. In the future, campaign chains will not be single-owner chains and should
+instead allow contributors (users with a pledge) to cancel the campaign if
+appropriate (even without the owner's cooperation).
+
+Optionally, contributors may also be able to create a block to accept a new epoch
+(i.e. a change of validators).
+
+# Usage
+
+## Setting Up
+
+The WebAssembly binaries for the bytecode can be built and published using [steps from the
+book](https://linera-io.github.io/linera-documentation/getting_started/first_app.html),
+summarized below.
+
+First, setup a local network with two wallets, and keep it running in a separate terminal:
+
+```bash
+./scripts/run_local.sh
+```
+
+Compile the Wasm binaries for the two applications `fungible` and `crowd-funding`
+application, and publish them as an application bytecode:
+
+```bash
+alias linera="$PWD/target/debug/linera"
+export LINERA_WALLET1="$PWD/target/debug/wallet.json"
+export LINERA_STORAGE1="rocksdb:$(dirname "$LINERA_WALLET1")/linera.db"
+export LINERA_WALLET2="$PWD/target/debug/wallet_2.json"
+export LINERA_STORAGE2="rocksdb:$(dirname "$LINERA_WALLET2")/linera_2.db"
+
+(cargo build && cd examples && cargo build --release)
+linera --wallet "$LINERA_WALLET1" --storage "$LINERA_STORAGE1" publish-bytecode \
+  examples/target/wasm32-unknown-unknown/release/fungible_{contract,service}.wasm
+linera --wallet "$LINERA_WALLET1" --storage "$LINERA_STORAGE1" publish-bytecode \
+  examples/target/wasm32-unknown-unknown/release/crowd_funding_{contract,service}.wasm
+```
+
+This will output two new bytecode IDs, for instance:
+
+```rust
+e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000
+e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000
+```
+
+We will refer to them as `$BYTECODE_ID1` and `$BYTECODE_ID2`.
+
+## Creating a Token
+
+In order to use the published bytecode to create a token application, the initial state must be
+specified. This initial state is where the tokens are minted. After the token is created, no
+additional tokens can be minted and added to the application. The initial state is a JSON string
+that specifies the accounts that start with tokens.
+
+In order to select the accounts to have initial tokens, the command below can be used to list
+the chains created for the test as known by each wallet:
+
+```bash
+linera --storage "$LINERA_STORAGE1" --wallet "$LINERA_WALLET1" wallet show
+linera --storage "$LINERA_STORAGE2" --wallet "$LINERA_WALLET2" wallet show
+```
+
+A table will be shown with the chains registered in the wallet and their meta-data:
+
+```rust
```

Comment:
`ignore` ?

Reply:
Ah, `cargo rdme` adds `rust` if you don't specify anything. I guess I'll make it `bash` then.

Reply:
Or `text` is probably more appropriate…